SimpleSession.py

```diff
@@ -109,10 +120,42 @@ def save(self, session):
         with open(session, 'w') as sess_file:
             json.dump(data, sess_file, indent=4)
 
+    def input_changed(self, session_name_prefix):
+        """
+            on input changed open autocomplete menu with a delay
+        """
+        if len(session_name_prefix) > 0 and \
+        self.input_panel and \
+        self.input_panel.command_history(0)[0] not in ['insert_completion', 'insert_best_completion']: #remove the looping
+            if self.input_panel.is_auto_complete_visible():
+                self.input_panel.run_command('hide_auto_complete')
+            delay = 500
```

Comment:
I'm not that familiar with this part of the Sublime API -- is there a reason for this delay?

Reply:
This delay is needed because without it the autocompletion menu will be shown while user is typing. 
Am I right that you want 'sublime.set_timeout(lambda: self.run_autocomplete(), 500)'? OK, I can do this. But it's a well known anti-pattern (https://en.wikipedia.org/wiki/Magic_number_(programming)), so I just have a reflex of making variables while writing code. :)

Reply:
> so I just have a reflex of making variables while writing code. :)

It's a good habit!


My general rule is that a "magic number" is okay if it's used just once; any more, and it gets a variable. Thanks for making the change!